scapy/arch/windows/__init__.py

```diff
@@ -945,7 +950,7 @@ def route_add_loopback(routes=None, ipv6=False, iflist=None):
         if iface.name == scapy.consts.LOOPBACK_NAME:
             IFACES.pop(devname)
     # Inject interface
-    IFACES[data['guid']] = adapter
+    IFACES["{0XX00000-X000-0X0X-X00X-00XXXX000XXX}"] = adapter
```

Comment:
What is this? Why?

Reply:
data["guid"] needed to be a real one for pcap stuff.
For routing, we don't mind.

Using IFACES[stuff] overwrites the working interface (that we used in data[guid]) with the one with fake routes, making network unreachable.

We only want to add another fake interface that will be used for special loopback routing only